lib/LocalVerifiableCredentialStore.js

```diff
@@ -18,12 +24,29 @@ export class LocalVerifiableCredentialStore {
     this.db.createIndex({
       index: {
         fields: [
-          'content.id',
-          'content.type',
-          'meta.parentId',
+          'meta.profileId',
+          'content.id'
+        ],
+        ddoc: 'getDoc'
+      }
+    });
+    this.db.createIndex({
+      index: {
+        fields: [
           'meta.profileId',
           'meta.displayable',
-        ]
+          'meta.parentId',
+        ],
+        ddoc: 'findDoc'
+      }
+    });
+    this.db.createIndex({
+      index: {
+        fields: [
+          'meta.profileId',
+          'content.type'
+        ],
+        ddoc: 'matchDoc'
```

Comment:
These names don't seem very meaningful to me. If you saw the names `getDoc`, `findDoc`, and `matchDoc` what would they mean to you? Would they clearly differentiate themselves from each other? If these are index names, then maybe they should be named after their fields:

`meta.profileId_content.id`, `meta.profileId_content.type`, `meta.profileId_meta.displayable_meta.parentId`

Or maybe they could be based on the type of query:

`byId`, `byType`, `byDisplay`/`byParent`

Or something else. The current naming convention doesn't seem to be helpful/intuitive -- and seems likely confusing/harmful.

I'm not going to hold the PR up for this -- but we should get it fixed soon as it will be a breaking change, presumably.